torch/_inductor/pattern_matcher.py

```diff
@@ -608,6 +608,279 @@ def addmm(match, mat1, mat2, inp):
         return L[aten.add](inp, L[aten.mm](mat1, mat2))
 
 
+# TODO(XiaobingSuper): move it to fx_passes/mkldnn_fuse.py
```

Comment:
I am ok with this stack of changes, but please move them into a separate file.

Reply:
You can create a `fx_passes/` folder to match with https://github.com/pytorch/pytorch/pull/97741

Reply:
Yes, move it to a separate file.